README.md

```diff
@@ -1,70 +1,6 @@
-# Getting Started with Create React App
+# This is a great README
```

Comment:
We will want this to probably be an h2 header and use an h1 header for the name of the app so:

```
# GRAPHQL-LEARN

## This is a great README
```